From 7b806a7e1d9eb373c22c8f4c6c9f218240636339 Mon Sep 17 00:00:00 2001
From: Dan Nica <dan.nica@nxp.com>
Date: Tue, 18 Aug 2020 09:51:11 +0300
Subject: [PATCH 171/269] s32g: Change FIP_MMC_OFFSET to 0x3400

With the adition of a duplicate IVT in the mkimage support
in u-boot, the FIP address on MMC has been shifted by 0x1000.

Issue: ALB-5434
Upstream-Status: Pending 

Signed-off-by: Dan Nica <dan.nica@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 plat/nxp/s32g/include/platform_def.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plat/nxp/s32g/include/platform_def.h b/plat/nxp/s32g/include/platform_def.h
index b5d32ebea..4e73879b1 100644
--- a/plat/nxp/s32g/include/platform_def.h
+++ b/plat/nxp/s32g/include/platform_def.h
@@ -171,7 +171,7 @@
 /* Must be placed by mkimage starting with AppBootCode:Code, but
  * aligned to the block size of 512 bytes
  */
-#define FIP_MMC_OFFSET		(0x2400)
+#define FIP_MMC_OFFSET		(0x3400)
 
 /* FIXME value randomly chosen; should probably be revisited */
 #define PLATFORM_STACK_SIZE		0x4000
-- 
2.17.1

